_docs/architecture.md

```diff
@@ -78,6 +78,22 @@ The following are examples of some possible webhook flows and events. Please not
 
 You can obtain an up-to-date list of available webhook events using the [Webhook Events endpoint](https://api.transferzero.com/documentation/#/Webhooks/get-webhook-events) of our API.
 
+## Collections
+We have available several type of webhooks that you can subscribe to that are you may find useful when doing collections. These webhooks start with `payin_method.`
+
+Here are the webhoks available:
+* `payin_method.incomplete`: This webhook is sent out in case there is some missing details that we require you to sent in order to proceed
```

Comment:
```suggestion
* `payin_method.incomplete`: This webhook is sent out in case there are some missing details that we require you to send in order to proceed
```